docs/sources/datasources/aws-cloudwatch/query-editor/index.md

```diff
@@ -253,7 +253,7 @@ To enable cross-account observability, complete the following steps:
 
 1. Add [two API actions]({{< relref "../../aws-cloudwatch#cross-account-observability-permissions" >}}) to the IAM policy attached to the role/user running the plugin.
 
-Cross-account querying is available in the plugin through the `Logs` mode and the `Metric search` mode. Once you have it configured correctly, you'll see a "Monitoring account" badge displayed in the query editor header.
+Cross-account querying is available in the plugin through the `Logs`, `Metric search` and `Metric Insights` mode. Once you have it configured correctly, you'll see a "Monitoring account" badge displayed in the query editor header. To support cross-account query building in `Metric Insights` builder mode, the cloudwatchMetricInsightsCrossAccount feature toggle should be enabled.
```

Comment:
I think bold for most of these because they are UI elements: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold.

Then I've put the feature flag in inline code because it's a reference to configuration: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#code

```suggestion
Cross-account querying is available in the plugin through the **Logs**, **Metric search**, and **Metric Insights** modes.
After you have it configured, you'll see a **Monitoring account** badge in the query editor header.
To support cross-account query building in the **Metric Insight** builder mode, the `cloudwatchMetricInsightsCrossAccount` feature toggle should be enabled.
```

Reply:
Thanks a lot @jdbaldry, this looks great!